src/components/ebay-chip/index.marko

```diff
@@ -0,0 +1,19 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+$ const { renderBody, a11yDeleteButton, ...htmlAttributes } = input;
+
+<span class="chip" ...processHtmlAttributes(htmlAttributes)>
+    <span class="chip__text" id:scoped="text">
```

Comment:
Just a nit, I would call the id `chip` or `title`. 
Text is pretty ambiguous. 
Its fine to leave as is though if there are no other changes.